src/crypto/crypto_argon2.cc

```diff
@@ -0,0 +1,172 @@
+#include "crypto/crypto_argon2.h"
```

Comment:
I'm not the most experienced with C++ yet, so this may be an incorrect recommendation, but would it make more sense to move this to within the `#ifndef OPENSSL_NO_ARGON2` block?

Reply:
Code-wise it doesn't really matter because the header includes the same guard. It's more of a habit and other files didn't have a similar code pattern for me to follow 😅